rclcpp/test/executors/test_multi_threaded_executor.cpp

```diff
@@ -87,12 +88,13 @@ TEST_F(TestMultiThreadedExecutor, timer_over_take) {
 
         if (diff < PERIOD - TOLERANCE) {
           executor.cancel();
+          // Using `ASSERT_GT` instead of `FAIL` here, so the numbers are logged.
```

Comment:
Nit: I'm not really sure that this comment is necessary, but I'll leave it to you whether you think it is valuable.

Reply:
I was originally confused, as I commented in https://github.com/ros2/rclcpp/pull/907#discussion_r418649282.

But I think it was just me being more silly than usual, so I deleted the comment in https://github.com/ros2/rclcpp/pull/1105/commits/b013cea1d5778c3688ff5a082bc2866699babc50.